production_entry_app/production_entry_app/utils/die_tool_counter.py

```
@@ -9,8 +9,19 @@
 from frappe.query_builder.functions import CustomFunction
 from frappe.utils import cint, get_datetime, now_datetime
 
+from production_entry_app.production_entry_app.joint_production import (
+	is_joint_lh_rh_production,
+)
+
 
 def update_counter_for_stock_entry(doc, direction: int = 1) -> None:
```

Ruff (0.16.1)

[warning] Too many return statements (7 > 6)

(PLR0911)